cmd/copy.go

```diff
@@ -32,15 +33,15 @@ var copyCmd = &cobra.Command{
 		allocationObj, err := sdk.GetAllocation(allocationID)
 		if err != nil {
 			fmt.Println("Error fetching the allocation", err)
```

Comment:
maybe it is better to use panic?
```suggestion
panic(""Error fetching the allocation: "+err.Error())
```